package.json

```diff
@@ -22,12 +22,16 @@
     "test": "yarn lint && yarn format && git diff --quiet"
   },
   "peerDependencies": {
-    "@babel/eslint-parser": "^7.15.0",
-    "eslint": "^8.1.0",
+    "@babel/core": "7.16.0",
```

Comment:
Why was `@babel/core` added?

Reply:
It was a required `peerDependency` of `eslint-parser` IIRC. I will remove it if it's no longer needed when we switch to `@babel/preset-react`.

Reply:
```
warning " > @babel/eslint-parser@7.16.3" has unmet peer dependency "@babel/core@>=7.11.0".
warning " > @babel/preset-react@7.16.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-react > @babel/plugin-transform-react-display-name@7.16.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-react > @babel/plugin-transform-react-jsx@7.16.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-react > @babel/plugin-transform-react-jsx-development@7.16.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-react > @babel/plugin-transform-react-pure-annotations@7.16.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-react > @babel/plugin-transform-react-jsx > @babel/plugin-syntax-jsx@7.16.0" has unmet peer dependency "@babel/core@^7.0.0-0".
```
...happens when adding `@babel/preset-react`.

Reply:
Usually, we would add that to `devDependencies` here and leave the peer decision up to the consumer.

Reply:
By _that_ I mean `@babel/core`.

Reply:
Will remove it

Reply:
Done https://github.com/zendeskgarden/eslint-config/pull/181/commits/5385e0867a16a13cf17515cdad95392b2ca89dba